### PR TITLE
New version based on your comments

### PR DESCRIPTION
**Pull request by Nanfeizhilu**
Dear Professor Grigorios Tsoumakas,
Based on your commonts, there are several changes in my code.
I have added MLHNLabelCorrelationTest class in test packages. The classifier have been tested by functions in MultiLabelLearnerTestBase and passed several of them. Because the hypernetwork based approach uses evolutionary learning strategy, the predictions of classifiers with identical parameter setting and identical data set are not all the same. So the classifier could not pass the testSameInputSameOutput function. Besides, I have reviewed the code I pushed to the original repository and translate or delete Chinese comments. Lastly, the file .project have been removed and the name of the package been altered.

### CODE REVIEW

File: mulan/test/mulan/classifier/MultiLabelLearnerTestBase.java [hunk 1]
```diff
@@ -341,6 +341,7 @@ public void testBuild() throws Exception {
         MultiLabelInstances mlDataSet = DataSetBuilder.CreateDataSet(DATA_SET);
         MultiLabelLearnerBase learner = getLearner();
 
+
```
Comment (tsoumakas) [inline mulan/test/mulan/classifier/MultiLabelLearnerTestBase.java]: No need to update this file with an additional line

File: mulan/test/mulan/classifier/hypernet/MLHNLabelCorrelationTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+	protected MLHNLabelCorrelation learner;
+	
+	
+    @Override
+    protected MultiLabelLearnerBase getLearner() {
+        return learner;
+    }
+
```
Comment (tsoumakas) [inline mulan/test/mulan/classifier/hypernet/MLHNLabelCorrelationTest.java]: Needs a @Before

**Comment by Nanfeizhilu** [comment]
Dear

**Comment by Nanfeizhilu** [comment]
Dear Professor Grigorios Tsoumakas,
The new version with the altered random number generating strategy are passed test functions (sameInputsameOutput, testBuildWithNonSparse, testBuildWith10ClassLabels, testBuild_WithDifferentOrder ) successfully. The needless line in the MultiLabelLearnerTestBase.java file is removed and @Before is added in MLHNLabelCorrelationTest.java

**Comment by Nanfeizhilu** [comment]
Dear Professor Grigorios Tsoumakas,
The new version with altered strategy of random number generating passes the test functions (sameInputsameOutput, testBuildWithNonSparse, testBuildWith10ClassLabels, testBuild_WithDifferentOrder) successfully. In addition, the needless line in  MultiLabelLearnerTestBase.java file is removed and the "@Before" is added in MLHNLabelCorrelationTest.java file.

**Pull request closed without merging**